%!TEX TS-program = xelatex
%!TEX encoding = UTF-8 Unicode
\documentclass[11pt, a4paper]{./Awesome-CV/awesome-cv}
\geometry{left=3.1cm, top=1.8cm, right=3.1cm, bottom=1.8cm, footskip=.5cm}% Configure page margins with geometry
\usepackage{wallpaper}
\input{./personal_info.tex}

%               LETTER INFORMATION
%---------------------------------------------------------
%\position{Bioinformatics Summer Intern}
\letterdate{\today}
%\lettertitle{Application for Bioinformatics Summer Intern}
\recipient{Regeneron} %company name
{}
\letteropening{Dear Hiring Manager,}% How the letter is opened
\letterclosing{Thank you for your time and consideration.\\ Sincerely,}
%\letterenclosure[Attached]{Résumé}% Any enclosures with the letter
\ULCornerWallPaper{1}{CMULetterHead.pdf}
\begin{document}
\makecvheader[-0.22in]{R}{16pt}{0.05in}
\topskip0pt
\vspace{-0.1in}
\vspace*{\fill}
%\vspace{0.2in}
\makelettertitle %Print the title with above letter information

\begin{cvletter}
\hspace{8mm} I am an first year MSc. student in Computational Biology at Carnegie Mellon University applying for the Clinical Informaticsinternship.
\par \hspace{8mm}
Curating and cleaning data are often the most difficult and time consuming parts of any analytics pipeline, as the saying goes ``garbage in, garbage out''.
I encountered this first hand during an internship in the Michael Hoffman lab at the University of Toronto under the supervision of Post-Doc and placenta expert Samantha Wilson.
Despite the fact that I was working with public data from NCBI there was still a significant amount of curating which data to use as the placenta contains both fetal and maternal cells.
I spent time reading papers to see how they handled their samples and writing R scripts to filter out data that did not meet our inclusion criteria.
After curating the data I had about 20,000 gene expression profiles and preforming analyses looking differentially expressed genes, relevant GO term and survival outcomes among others.
I also visualized my results and presented them to computational and wet-lab scientists to better understand what the data was saying about the relationship between the placenta and cancer.
\par \hspace{8mm}
I also have some experience with pipeline development from my undergraduate thesis.
I was designing an end-to-end pipeline to explore the relationship between rates of horizontal gene transfer and the presence of CRISPR-Cas systems in bacteria.
I wrote scripts to retrieve the data for all complete bacterial genomes on NCBI, produce gene and species trees for each species and preform network analysis.
Since I was working with several thousand genomes I also needed to make sure that the code was relatively efficient and produced clear progress reports and organized output files.
This involved significant python, R and shell scripting, working with and generating structured and unstructured data in an organized and reproducible way.
\par \hspace{8mm}
My formal education in Genetics and Molecular biology in my undergrad was complimented by my work and extracurricular experience writing code and working with biological data.
I am now furthering my skills and knowledge by pursuing an MSc. in Computational Biology, shoring up my quantitative skills and gaining more experience with analyzing biological data.
I have completed/am currently taking courses in Computational Genomics, Statistics and Machine Learning as well as a practicum course building a bioinformatics pipeline.
I think I would be a great fit for this clinical informatics internship and would love to apply my skills to such an interesting, important problem.

\end{cvletter}
\makeletterclosing
\vspace*{\fill}
\makecvfooter{Siddharth Reed}{}{Cover Letter} %Print the footer with 3 arguments(<left>, <center>, <right>)}
\end{document}

